app/models/config.rb

```diff
@@ -343,38 +374,45 @@ def validate_start_of_week
     }.stringify_keys!
 
     def validate_time_zone
-      TIME_ZONE.keys.include?(options.last.titleize)
+      unless TIME_ZONE.keys.include?(options.last.titleize)
+        "Timezone provided is not supported"
+      end
     end
 
     ### Practical support
 
     def validate_yes_or_no
       # allow yes or no, to be nice (technically only yes is considered)
-      options.last =~ /\A(yes|no)\z/i
+      if !(options.last =~ /\A(yes|no)\z/i)
```

Comment:
made a very slight adjustment to this to solve for an order-of-operations problem -- `!options.last =~ regex` was casting options.last to a boolean and then trying to string match that, so it was causing failures. Wrapping in parens solves for it